apiclient/templates/typescript/axios/generate.sh

```diff
@@ -4,7 +4,7 @@ artifactsDirectory="${1}"
 buildNumber="${2}"
 if [[ -n ${buildNumber} ]]; then
     # Unstable build
-    additionalProperties=",snapshotVersion=\"-SNAPSHOT.${buildNumber}\",npmRepository=\"https://pkgs.dev.azure.com/jellyfin-project/jellyfin/_packaging/unstable/npm/registry/\""
+    additionalProperties=",snapshotVersion='-SNAPSHOT.${buildNumber}',npmRepository='https://pkgs.dev.azure.com/jellyfin-project/jellyfin/_packaging/unstable/npm/registry/'"
```

Comment:
```suggestion
    additionalProperties=",snapshotVersion=-SNAPSHOT.${buildNumber},npmRepository=https://pkgs.dev.azure.com/jellyfin-project/jellyfin/_packaging/unstable/npm/registry/"
```

After testing- the only way this works is if there are no quotes.